File: module-common-adapter/src/main/java/com/common/adapter/utils/MockUtils.java
package com.common.adapter.utils;

import java.util.ArrayList;
import java.util.List;

public class MockUtils {

    public List<String> getMockStringList(int size) {
        List<String> stringList = new ArrayList<>();
        String content = "mock data";
        for (int i = 0; i < size; i++) {
            stringList.add(i + "--->" + content);
        }
        return stringList;
    }
}

File: module-common-adapter/src/main/java/com/common/adapter/utils/MockUtils.kt
package com.common.adapter.utils

import java.util.ArrayList

object MockUtils {

    fun getMockStringList(size: Int): List<String> {
        val stringList: MutableList<String> = ArrayList()
        val content = "mock data"
        for (i in 0 until size) {
            stringList.add("$i--->$content")
        }
        return stringList
    }

}